### DISCUSSION TITLE
Copilot icon

### DISCUSSION THREAD

**Discussion by Unkrown6YT** [valinet/ExplorerPatcher#2501] [Ideas]
Since 10.0.19045.3757 was released, Windows Copilot has been added to Windows 10 (Release Preview Channel). Copilot should be added to the Windows 10 taskbar style. When the Copilot icon is disabled in settings, the icon will disappear. It should be on the right or left

**Comment by Amrsatrio** [comment]
I'm still waiting until the symbols for 3757 have been made available, from that point I will begin to backport the copilot button.

**Comment by brtbendele** [comment]
I can't get my Copilot button to reappear.  It appears for one session every time Windows Preview does an update.

**Comment by zake2002v2** [comment]
It's been officially added in Windows 10, [located where the notification button should be](https://github.com/valinet/ExplorerPatcher/assets/42307974/41a6f42b-03ff-4965-bab9-4815f4d6f8c2), it seems to push the notifications button to the side
If I may add to the suggestion, it should instead replace the no longer working Cortana button! Probably as another option in the explorer patcher configs
[Actually, on second thought, I'll just make that into its own separate idea](https://github.com/valinet/ExplorerPatcher/discussions/2948)

**Comment by Amrsatrio** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/823285be-fc6f-4673-a474-08bd9e462425)
![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/a6b7ee26-6be5-4443-9098-675510432fd2)
![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/6bce2f92-0451-4743-8f7b-34fbf9fe85cf)

Releasing hopefully next month.

**Comment by YKefasu** [comment]
https://github.com/valinet/ExplorerPatcher/issues/2866#issuecomment-2019126665

I found a solution to do this as a temporary solution:
![image](https://github.com/valinet/ExplorerPatcher/assets/129555908/78b7af47-f07e-44ba-8157-d3ccdd931234)

1. Desktop > right click > new > shortcut
2. Browse to location of edge browser
3. Once the shortcut is made, right click > properties
4. In the target field, just add `microsoft-edge://?ux=copilot&tcp=1&source=taskbar` after the target exe
it should be like ` "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe" microsoft-edge://?ux=copilot&tcp=1&source=taskbar `

![image](https://github.com/valinet/ExplorerPatcher/assets/129555908/518db3cb-64cc-4c5b-b667-65e1281faea3)

5. change the icon to copilot 

![image](https://github.com/valinet/ExplorerPatcher/assets/129555908/97f5cae1-6248-419e-a084-b569b1a708bd)

7. Right click shortcut > more options > pin to taskbar

![image](https://github.com/valinet/ExplorerPatcher/assets/129555908/52717fda-4a93-4b67-9865-cb44fd79417f)
